src/components/header-banner/header-banner.jsx

```diff
@@ -4,23 +4,20 @@ import { HeaderCircleIcon } from "../icons";
 
 import * as styles from "./header-banner.module.scss";
 
-const repeatCategories = () => {
-  for (let i = 0; i < 10; i++) {
-    return (
-      <div className={styles.bottomTextSubContainer}>
-        <span>Development</span>
-        <span>•</span>
-        <span>Design</span>
-        <span>•</span>
-        <span>Machine Learning</span>
-        <span>•</span>
-        <span>Blockchain</span>
-        <span>•</span>
-        <span>People</span>
-      </div>
-    )
-  }  
-}
+const repeatCategories = () => (
```

Comment:
The name of the function is not longer correct

Reply:
Done